projects/systelab-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "17.1.11",
+  "version": "17.1.12",
```

Comment:
It would be better to move to 17.2.0 because this change can't be considered a patch

Reply:
I think it is correct he is undoing a breaking change he introduced by mistake